package.json

```diff
@@ -66,6 +66,7 @@
     "@rollup/plugin-node-resolve": "^6.1.0",
     "asyncro": "^3.0.0",
     "autoprefixer": "^9.7.3",
+    "babel-jest": "^24.8.0",
```

Comment:
I moved this to `dependencies` because it is a dependency of code that the package exports (`microbundle-jest.js`).